lib/CL/devices/hsa/pocl-hsa.c

```diff
@@ -118,6 +126,112 @@ pocl_hsa_get_agents(hsa_agent_t agent, void *data)
   return HSA_STATUS_SUCCESS;
 }
 
+//Get hsa-unsupported device features from hsa device list.
+int num_hsa_device = 1;
+hsa_device_features hsa_device_lists[MAX_HSA_AGENTS];
+void init_hsa_device_list()
+{
+	hsa_device_lists[0].dev_name = "Spectre";
```

Comment:
Please check the indentation here. We use spaces instead of tabs and 2 spaces per indentation. The C codes should aim to follow the GNU C style, C++ closer to LLVM's.